projects/ui-components/lf-repository-browser/ILfTreeNodeService.ts

```diff
@@ -15,28 +15,28 @@ export interface LfTreeNodePage {
 export interface LfTreeNodeService {

```

Comment:
/**
A tree data structure provider that can be used as a data source for UI components, such as the lf-repository-browser component,  to navigate a tree structure.
*/